database/src/migrations/20221209131600_validation_rule_unique_error.ts

```diff
@@ -0,0 +1,46 @@
+import { Knex } from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+const summaryMessageTypes = [
+  {
+    name: 'Non Unique Key found',
```

Comment:
I think this string needs to match the enum value defined in `constraints/status.ts`, since we search the database using the name of an error in order to find its ID when inserting a submission error.